index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap" rel="stylesheet">
+  
+  <link rel="stylesheet" href="style.css">
+  <title>Aliza Amir Portfolio</title>
+</head>
+<body>
+  <nav>
+    <h3 class="logo">Aliza Amir</h3>
+    <span class="icon">&#9776</span>
+  </nav>
+  <header>
+<h1 class="header-text">
+  I am Aliza. <br> glad to see <br>you!
+</h1>
+<p class="paragraph">
+  I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h4 class="connect">LET'S CONNECT</h4>
+<ul class="icons">
+  <li class="twitter"><img src="./images/twitter.png" ></li>
+  <li class="linkedln"><img src="./images/Linkedin.png" alt=""></li>
+  <li class="twitter"><img src="./images/gmail.png" alt=""></li>
+  <li class="twitter"><img src="./images/github.png" alt=""></li>
+  <li class="twitter"><img src="./images/Vector.png" alt=""></li>
+
+</ul>
```

Comment:
- [ ] Good to have your social media links use a `ul` tag to group them, and `li` per icon; But make sure each icon has a link (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href Like this
  ```html
   <ul>
        <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
        <!-- other links here -->
   </ul>